<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWin</class>
 <widget class="QWidget" name="RegisterWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_19" stretch="1,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0,0,0,0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>姓名：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid black; border-radius: 10px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,8">
         <item>
          <widget class="QLabel" name="wrb_3">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>性别：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sex_comboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>10</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">min-width: 150px;  /* 设置最小宽度 */  
min-height: 30px; /* 设置最小高度 */  
font-size: 15px;  /* 设置字体大小 */  
border: 1px solid black;
border-radius:10px;</string>
           </property>
           <item>
            <property name="text">
             <string>请选择</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>男</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>女</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>学号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="numberEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid black; border-radius: 10px;
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>输入12位学号</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>学院：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="school_comboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>10</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">min-height: 30px; /* 设置最小高度 */  
font-size: 15px;  /* 设置字体大小 */  
border: 1px solid black;
border-radius:10px;
QComboBox QAbstractItemView {  
    min-width: 300px; /* 设置下拉列表的最小宽度 */  
    font-size: 14px;  /* 设置下拉列表的字体大小，可选 */  
}</string>
           </property>
           <item>
            <property name="text">
             <string>请选择</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>机械与汽车工程学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>马克思主义学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>建筑学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>土木与交通学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>电力学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>电子与信息学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>材料科学与工程学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>化学与化工学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>轻工科学与工程学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>食品科学与工程学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>数学学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>物理与光电学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>工商管理学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>外国语学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>计算机科学与工程学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>自动化科学与工程学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>体育学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>软件学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>国际教育学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>医学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>经济与贸易学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>环境与能源学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>生物科学与工程学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>新闻与传播学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>艺术学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>法学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>设计学院</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>宿舍号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="dormitoryEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid black; border-radius: 10px;</string>
           </property>
           <property name="placeholderText">
            <string>输入格式如：C1-101</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>电话：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="phone_number_Edit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid black; border-radius: 10px;</string>
           </property>
           <property name="placeholderText">
            <string>输入11位电话号码</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <item>
          <widget class="QPushButton" name="registerBt">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>注册</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="resetBt">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>重置</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,0">
       <item>
        <widget class="QLabel" name="headpicLb">
         <property name="styleSheet">
          <string notr="true">border-image: url(:/addpic.png);
background-color: rgb(0, 0, 0);
background-color: rgb(240, 240, 240);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="picfileEdit">
         <property name="placeholderText">
          <string>图像路径</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="addpicBt">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>添加头像</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="videoswitchBt">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>打开摄像头</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cameraBt">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>拍照</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
